UniAtHome/UniAtHome.BLL/Services/UniversityCreationService.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UniAtHome.BLL.DTOs.UniversityCreation;
+using UniAtHome.BLL.Exceptions;
+using UniAtHome.BLL.Interfaces;
+using UniAtHome.DAL.Entities;
+using UniAtHome.DAL.Interfaces;
+
+namespace UniAtHome.BLL.Services
+{
+    public sealed class UniversityCreationService : IUniversityCreationService
+    {
+        private readonly IRepository<UniversityCreateRequest> requestsRepository;
+
+        public UniversityCreationService(IRepository<UniversityCreateRequest> requestsRepository)
+        {
+            this.requestsRepository = requestsRepository;
+        }
+
+        public async Task AddRequestAsync(UniversityCreateDTO creationInfo)
+        {
+            var createRequest = new UniversityCreateRequest
+            {
+                UniversityName = creationInfo.UniversityName,
+                SubmitterFirstName = creationInfo.SubmitterFirstName,
+                SubmitterLastName = creationInfo.SubmitterLastName,
+                Email = creationInfo.Email,
+                Comment = creationInfo.Comment,
+                Submitted = DateTime.Now
```

Comment:
UniversityCreateRequest object is inherited from baseauditable entity.. Do u need this field?

Reply:
It doesn't inherit auditable entity - [look](https://github.com/arstrybuh257/UniAtHome.API/blob/6beb0898d38922d91685d55eeb92c6c793041dc5/UniAtHome/UniAtHome.DAL/Entities/UniversityCreateRequest.cs#L5). Nobody is going to change these records after their creation. And yes, we need this field in order to sort requests on the web page by their time.